editions/tw5.com/tiddlers/nodejs/TiddlyWiki on Node.js.tid

```diff
@@ -1,15 +1,20 @@
 created: 20131129094353704
-modified: 20200318115527226
+modified: 20220612092156888
 tags: Platforms
 title: TiddlyWiki on Node.js
 type: text/vnd.tiddlywiki
 
+\rules except wikilink
+
 Running TiddlyWiki on [[Node.js]] brings several important benefits over and above the single file version:
 
-* You can edit your content on any suitably compatible HTML5 browser, including smartphones and tablets
 * Individual tiddlers are stored in separate files, which you can organise as you wish
+
 * The ability to build multiple wikis that blend different combinations of shared and unique content
 
+* You can edit your content on any suitably compatible HTML5 browser, including smartphones and tablets
```

Comment:
Perhaps better to say "any modern browser". HTML5 isn't really a contemporary term.